# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/fetch/InternalRequest.cpp
# Commit: 66b279e5a513
# Full Hash: 66b279e5a513bf706129ac64cf44e3353e3a0d71
# Author: Eden Chuang <echuang@mozilla.com>
# Date: 2023-01-12 21:30:33
# Regressor Bug: 1351231
# File Overlap Count: 1
# Description:
#   Bug 1351231 - FetchService integration for PFetch. r=dom-worker-reviewers,jesup
#   
#   Depends on D142436
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D142437
# ==============================================================================

diff -r 3eb8fdd0ba6d -r 66b279e5a513 dom/fetch/InternalRequest.cpp
--- a/dom/fetch/InternalRequest.cpp	Thu Jan 12 17:24:28 2023 +0000
+++ b/dom/fetch/InternalRequest.cpp	Thu Jan 12 17:24:28 2023 +0000
@@ -193,7 +193,7 @@
   if (!XRE_IsParentProcess()) {
     if (body) {
       MOZ_ASSERT(body->type() == BodyStreamVariant::TParentToChildStream);
-      mBodyStream = body->get_ParentToChildStream().stream();
+      mBodyStream = body->get_ParentToChildStream().get_RemoteLazyInputStream();
     }
   } else {
     if (body) {